.eleventy.js

```diff
@@ -295,6 +297,71 @@ module.exports = function (eleventyConfig) {
     },
   );
 
+  /*
+   * Shortcode to render demo dates
+   * Renders based on data file at ./src/_data/registerdemos.js
+   */
+  eleventyConfig.addPairedShortcode(
+    'registerForDemoLinks',
+    (children, locale) => {
+      const langStrings = {
+        en: {
+          enDemo: 'English demo',
+          frDemo: 'French demo',
+          timezone: 'Eastern time',
+        },
+        fr: {
+          enDemo: 'Démonstration en anglais',
+          frDemo: 'Démonstration en français',
```

Comment:
Please change "Démonstration en français" to "Démo en français"

Reply:
I as able to commit the suggestions. Change "démonstration" to "démo" throughout.